code-of-conduct.md

```diff
@@ -1,5 +1,15 @@
 ## CNCF Community Code of Conduct v1.0
 
+Other languages available:
+- [Chinese](code-of-conduct-languages/zh.md)
+- [German](code-of-conduct-languages/de.md)
+- [Spanish](code-of-conduct-languages/es.md)
+- [French](code-of-conduct-languages/fr.md)
+- [Italian](code-of-conduct-languages/it.md)
+- [Japanese](code-of-conduct-languages/jp.md)
+- [Korean](code-of-conduct-languages/ko.md)
+- [Ukrainian](code-of-conduct-languages/uk.md)
```

Comment:
Can you please edit to put so that it does "German / Deutch" like in the language switcher at the top right of https://bestpractices.coreinfrastructure.org/en

Reply:
@dankohn only for German or all languages?